src/common/grpc/src/channel_manager.rs

```diff
@@ -261,6 +262,8 @@ pub struct ChannelConfig {
     pub max_recv_message_size: ReadableSize,
     // Max gRPC sending(encoding) message size
     pub max_send_message_size: ReadableSize,
+    // Enable gzip compression for gRPC
```

Comment:
Maybe:
```suggestion
    // Supported compression encoding for gRPC, e.g: gzip, zstd.
```
If I understand this code correctly.